modules/virt-customizing-storage-profile.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc
+
+[id="virt-customizing-storage-profile_{context}"]
+
+= Customizing the storage profile
+
+Customize your storage profile to define default settings for data volumes that you frequently allocate. You can create as many storage profiles as you want, associating each with a storage class. 
```

Comment:
You cannot create storage profiles.  Their lifecycle is managed by CDI according to which storage classes are in the system

Reply:
So you can create multiple storage classes and the associated storage profile will be created automatically.

Reply:
Yeah, I finally got that :-)